src/applications/gi/components/search/EligibilityForm.jsx

```diff
@@ -9,6 +9,7 @@ import Dropdown from '../Dropdown';
 
 import recordEvent from 'platform/monitoring/record-event';
 import { isLoggedIn } from 'platform/user/selectors';
+import { eBenefitsUrlGenerator } from 'platform/utilities/eBenefitsUrl';
```

Comment:
Can you also update https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/components/profile/BenefitsForm.jsx (newer version of this component behind `gibctEstimateYourBenefits` feature flag) as well?

Reply:
sure thing